CHANGELOG.md

```diff
@@ -15,12 +15,74 @@ For reference, the possible headings are:
 * **Notes**
 
 
-## [0.6.0] - 2016-09-01
-Tag name: v0.6.0
+## [0.7.0] - 2016-10-28
+Tag name: v0.7.0
 = commit: 
```

Comment:
The commit will be the one that gets the v0.7.0 tag (git tag), which is a tag that GitHub creates when we create the GitHub "release." It's usually the commit that bumped the version number, because that's the last thing we do before creating a release.